src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -221,10 +221,10 @@ const PersonalDetails = (() => {
                     return;
                 }
                 const get_settings    = data.get_settings;
-                const has_required_mt = (/real_vanuatu_(standard|advanced)/.test(redirect_url) ?
-                    (get_settings.tax_residence && get_settings.tax_identification_number && get_settings.citizen)
+                const has_required_mt = (+State.getResponse('landing_company.config.tax_details_required') === 1 ?
+                    (get_settings.tax_residence && get_settings.tax_identification_number)
                     :
-                    get_settings.citizen // only check Citizen if user selects mt volatility account
+                    get_settings.citizen
```

Comment:
I'm confused, now we are only checking the current binary accounts needs tax details, but as per the MT5 check there are binary accounts that don't need tax details but for those MT5 account openings they need tax details. doesn't this break that check, or am I missing something?

Reply:
i've checked with QA and compliance, the same applies to MT5 account openings as well, we need to ask for tax info only if their country require tax info (check for `tax_details_required`), and citizen is needed for all MT5 accounts